R/guide-old.R

```diff
@@ -90,7 +90,7 @@ GuideOld <- ggproto(
   train = function(self, params, scale, aesthetic = NULL,
                    title = waiver(), direction = NULL) {
     params$title <- scale$make_title(params$title %|W|% scale$name %|W|% title)
-    params$direction <- params$direction %||% direction
+    params$direction <- params$direction %||% direction %||% "vertical"
```

Comment:
For old guides, their methods may still expect `direction` to be available during training, which it is not anymore.
To preserve *most* of their functionality, we're defaulting their direction to vertical, which is probably 80% of plots.